<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>脊椎角度估算</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPlainTextEdit" name="file_msg">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="backgroundVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="choose_file">
        <property name="text">
         <string>选择图片</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="5,1">
      <item row="0" column="0">
       <widget class="QLabel" name="origin_img">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="result_image">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="show_text">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="backgroundVisible">
         <bool>false</bool>
        </property>
        <property name="centerOnScroll">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>choose_file</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>on_choose_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>612</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_choose_file()</slot>
 </slots>
</ui>
